public/scripts/controllers/scoreController.js

```diff
@@ -0,0 +1,20 @@
+angular.module('scrambleApp').
+  controller('scoreController', ['$scope', 'gameStateService',
+             function($scope, gameStateService) {
+    //Display the score at the end of the game, allows the user to restart the game
+
+    var currentState = gameStateService.getState();
+    $scope.score = currentState.score;
```

Comment:
expose a getter on the gameStateService for score and word, then you can do...

  $scope.score = currentState.getScore;